README.md

```diff
@@ -1,5 +1,13 @@
 # search-a-tags
 
+[![NPM Version](https://img.shields.io/npm/v/search-a-tags-hacktoberfest.svg)](https://www.npmjs.com/package/search-a-tags-hacktoberfest)
+[![NPM Downloads](https://img.shields.io/npm/dt/search-a-tags-hacktoberfest.svg)](https://www.npmjs.com/package/search-a-tags-hacktoberfest)
+[![Github Repo Size](https://img.shields.io/github/repo-size/AryanAg08/search-a-tags-hacktoberfest.svg)](https://github.com/AryanAg08/search-a-tags-hacktoberfest)
+[![LICENSE](https://img.shields.io/npm/l/search-a-tags-hacktoberfest.svg)](https://github.com/AryanAg08/search-a-tags-hacktoberfest/blob/master/LICENSE)
+[![Contributors](https://img.shields.io/github/contributors/AryanAg08/search-a-tags-hacktoberfest.svg)](https://github.com/AryanAg08/search-a-tags-hacktoberfest/graphs/contributors)
+[![Commit](https://img.shields.io/github/last-commit/AryanAg08/search-a-tags-hacktoberfest.svg)](https://github.com/AryanAg08/search-a-tags-hacktoberfest/commits/master)
+
+
 [![Build](https://github.com/arshadkazmi42/search-a-tags/actions/workflows/nodejs.yml/badge.svg)](https://github.com/arshadkazmi42/search-a-tags/actions/workflows/nodejs.yml)
```

Comment:
Can you move this to line 3, we want build badge to be the first badge in the list